### PR TITLE
SB-517+SB-732

### PR DESCRIPTION
**Pull request by sbespalov**
SB-732: UserRoles fixed
SB-715:  "keep me signed in for x" amount of time Implemented

### CODE REVIEW

File: strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/UserController.java [hunk 1]
```diff
@@ -248,19 +248,16 @@ public ResponseEntity generateSecurityToken(@ApiParam(value = "The name of the u
...
     }
 
-    @ApiOperation(value = "Generate authentication token.",
-                  position = 3)
-    @ApiResponses(value = { @ApiResponse(code = 200,
-                                         message = "The aouthentication token was generated."),
-                            @ApiResponse(code = 500,
-                                         message = "An error occurred.") })
+    @ApiOperation(value = "Generate authentication token.", position = 3)
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "The aouthentication token was generated."),
```
Comment (carlspring) [inline strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/UserController.java]: There's a typo -- `authentication`, not `aouthentication`.

File: strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java [hunk 2]
```diff
@@ -32,6 +33,12 @@
         extends WebSecurityConfigurerAdapter
 {
 
+    @Configuration
+    @EnableGlobalMethodSecurity(prePostEnabled=true)
+    public static class MethodSecurityConfig extends GlobalMethodSecurityConfiguration {
+
```
Comment (sbespalov) [inline strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java]: this is SB-732 fix, tjis configuration enables `@PreAuthorize` annotations

Comment (carlspring) [inline strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java]: Riight... I see... Could you just move the opening brace on a new line, please?

File: strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/UserController.java (hunk 1 above)
Comment (sbespalov) [inline strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/UserController.java]: fixed

File: strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java (hunk 2 above)
Comment (sbespalov) [inline strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java]: done

**Pull request merged**